src/modules/create_cards.js

```diff
@@ -0,0 +1,58 @@
+const createCards = async (cards) => {
+  const response = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/hrqIdiXTh94rmLQMrXcG/likes/');
+  const data = await response.json();
+
+  // console.log(cards);
```

Comment:
- [OPTIONAL] KIndly Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data visible to anyone using the browser tools to look at your website. 💯 🧑‍🔧